--- a/src/main/java/io/papermc/paper/ServerBuildInfo.java
+++ b/src/main/java/io/papermc/paper/ServerBuildInfo.java
@@ -17,6 +_,13 @@
      */
     Key BRAND_PAPER_ID = Key.key("papermc", "paper");
 
+    // Flint start - brand
+    /**
+     * The brand id for FlintMC.
+     */
+    Key BRAND_FLINT_ID = Key.key("flintmc", "flint");
+    // Flint end - brand
+
     /**
      * Gets the {@code ServerBuildInfo}.
      *
