--- a/src/main/java/io/papermc/paper/ServerBuildInfo.java
+++ b/src/main/java/io/papermc/paper/ServerBuildInfo.java
@@ -17,6 +_,13 @@
      */
     Key BRAND_PAPER_ID = Key.key("papermc", "paper");
 
+    // Ember start - brand
+    /**
+     * The brand id for EmberMC.
+     */
+    Key BRAND_EMBER_ID = Key.key("embermc", "ember");
+    // Ember end - brand
+
     /**
      * Gets the {@code ServerBuildInfo}.
      *
